core/unit_test/neighbor_unit_test.hpp

```diff
@@ -76,13 +76,23 @@ Cabana::AoSoA<Cabana::MemberTypes<double[3]>, TEST_MEMSPACE>
 createParticles( const int num_particle, const double box_min,
                  const double box_max )
 {
-    using DataTypes = Cabana::MemberTypes<double[3]>;
-    using AoSoA_t = Cabana::AoSoA<DataTypes, TEST_MEMSPACE>;
-    AoSoA_t aosoa( "aosoa", num_particle );
+    #ifdef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
```

Comment:
Will this not make the test spaces always use the host when OpenMP Target is used even when another backend may also be enabled in the build? Maybe just disable this test for now when the backend is OpenMP Target? You would need to use some type traits to do that effectively.

Reply:
Or is the issue random number generation isn't implemented? You could just build them on the host and copy them to whatever device type then to get this test to work.

Reply:
Yeah, this is just temporarily switched to the host. I switched all random generation to the host first - I can go back to that to make it cleaner